app/assets/stylesheets/application.css.scss

```diff
@@ -15,46 +15,50 @@
  *= require font-awesome
  */
 
- .social-links a {
-   text-align: center;
-   display: inline-block;
-   width: 36px;
-   height: 36px;
-   border: 1px solid white;
-   border-radius: 100%;
-   margin-right: 7px; /*space between*/
- }
-
- .social-links a i {
-   font-size: 20px;
-     line-height: 38px;
-   color: white;
- }
-
- #facebook {
-     background: #3B5998;
- }
-
- #twitter {
-     background: #6CDFEA;
- }
-
- #instagram {
-     background: #e74c3c;
- }
+* {
+  font-family: "Open Sans";
+}
+
+.social-links a {
+  text-align: center;
+  display: inline-block;
+  width: 36px;
+  height: 36px;
+  border: 1px solid white;
+  border-radius: 100%;
+  margin-right: 7px; /*space between*/
+}
+
+.social-links a i {
+  font-size: 20px;
+  line-height: 38px;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.